CarShop/src/main/java/pl/Lukasz/CarShop/Car/CarController.java

```diff
@@ -17,7 +17,7 @@ public class CarController {
 
     private final CarService carService;
 
-    @GetMapping("/carbyid/{id}")
+    @GetMapping("/car")
```

Comment:
```suggestion
@GetMapping("/car"/{id})
```

Nie brakuje tu `id` w pathie?